packages/data-service/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "data-service",
+  "module": "./src/index.js",
+  "version": "0.0.1",
```

Comment:
usually semver doesn't start with a bugfix but it's ok :)

Reply:
Ubuntu first bug:
https://bugs.launchpad.net/ubuntu/+bug/1